chandra_aca/aca_image.py

```diff
@@ -828,3 +832,82 @@ def get_psf_image(
 
         out = ACAImage(psf, row0=row0, col0=col0) if aca_image else (psf, row0, col0)
         return out
+
+
+@retry.retry(exceptions=requests.exceptions.RequestException, delay=5, tries=3)
+def get_aca_image_table(
```

Comment:
Is this kinda what you meant @taldcroft ?  That I could have my do-all function to get images and background-subtracted versions but you wanted the to keep it as a separate function in aca_image?